README.md

```diff
@@ -58,234 +17,92 @@ After you're finished please remove all the comments and instructions!
 - [🤝 Contributing](#contributing)
 - [⭐️ Show your support](#support)
 - [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ](#faq)
 - [📝 License](#license)
 
-<!-- PROJECT DESCRIPTION -->
-
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖Vet_Clinic_Database
 
-> Describe your project in 1 or 2 sentences.
+<a name="about-project"></a>
 
-**[your_project__name]** is a...
-
-## 🛠 Built With <a name="built-with"></a>
+# Vet_Clinic_Database
+This is a PostgreSQL vet clinic database create to make user to access animal database.
 
-### Tech Stack <a name="tech-stack"></a>
 
-> Describe the tech stack and include only the relevant sections that apply to your project.
+![screenshot](![Screen Shot 2023-05-16 at 16 23 46](https://github.com/TaiwoEnoch/Vet_clinic_database/assets/111912283/b1fef055-5fe3-4325-8b97-04a8aeef193e))
 
-<details>
-  <summary>Client</summary>
-  <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
-  </ul>
-</details>
-
-<details>
-  <summary>Server</summary>
-  <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
-  </ul>
-</details>
+<!-- LIVE DEMO -->
 
-<details>
-<summary>Database</summary>
-  <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
-  </ul>
-</details>
+## 🚀 Live Demo <a name="live-demo"></a>
 
-<!-- Features -->
+- [To be unpdated]
```

Comment:
- [ ] [OPTIONAl] I would suggest to remove the live demo section from the Readme since it is a database project and there is no live version of it  :+1: